src/Shared/Compat/parametermodifier.cs

```diff
@@ -13,8 +13,9 @@ namespace System.Reflection
 {  
     using System;
 
+
```

Comment:
I meant to revert this file.  This is now done and I force pushed to get rid of this change.